cartridges/int_cloudinary/cartridge/scripts/util/cloudinaryConstants.js

```diff
@@ -127,6 +127,7 @@ CloudinaryConstants.CLD_LIST_SERVICE_API_KEY_PLACEHOLDER = '[apikey]';
 CloudinaryConstants.CLD_LIST_SERVICE_API_SECRET_PLACEHOLDER = '[apisecret]';
 CloudinaryConstants.CLD_UPLOAD_SVC = 'cloudinary.rest.upload';
 CloudinaryConstants.CLD_LIST_SVC = 'cloudinary.rest.list';
+CloudinaryConstants.CLD_METADATA_UPLOAD = 'cloudinary.metadata.upload';
```

Comment:
The metadata belongs to the uploaded assets, thus, no need for a new service.
Please use the CLD_UPLOAD_SVC